README.md

```diff
@@ -119,6 +119,15 @@ docker run --rm -i -p 3333:3333 -e QUICKPIZZA_OTLP_ENDPOINT=http://grafana-agent
 
 The `QUICKPIZZA_TRUST_CLIENT_TRACEID` environment variable instructs QuickPizza to trust client-provided Trace IDs, so all client-generated spans (if any) will show up in the trace.
 
+## Enable Grafana Faro
+
+To enable Grafana Faro and monitor frontend observability, simply set `QUICKPIZZA_CONF_FARO_URL` to your Faro web URL:
```

Comment:
```suggestion
To enable Grafana Faro and monitor frontend observability, stop the QuickPizza container if it's already running and simply set `QUICKPIZZA_CONF_FARO_URL` to your Faro web URL:
```

Reply:
👍🏻 
I've also removed the "simply", as I like to follow the https://justsimply.dev/ advice. We can bring it back if there are stronger editorial reasons to use it.